SQL-Scripts/create_German_ref_data.sql

```diff
@@ -0,0 +1,64 @@
+/* Change Log															*/
+/* Beta - 4/4/2021 - Garrett Reed - Initial Creation - Create German Reference Tables and Data	*/
+
+use wordsense;
+
+/* Create table structures */
+DROP TABLE IF EXISTS German_Index;
+DROP TABLE IF EXISTS German_Corpus;
```

Comment:
Good! In case the table is ￼corrupt

Reply:
Looks Good! We can add to check status of table to handle if table corrupted